src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -1660,38 +1661,45 @@ CHIP_ERROR ValidateCertificateChain(const uint8_t * rootCertificate, size_t root
     X509 * x509CACertificate   = nullptr;
     X509 * x509LeafCertificate = nullptr;
 
+    VerifyOrReturnError(rootCertificate != nullptr && rootCertificateLen != 0,
```

Comment:
Start by setting result to `kInternalFrameworkError`to guard for unexpectd paths

Reply:
Fixed in 45c4216